src/Extend/Console.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Illuminate\Contracts\Container\Container;
+
+class Console implements ExtenderInterface
+{
+    protected $addCommands = [];
+
+    /**
+     * Add a command to the console.
+     *
+     * @param string $command ::class attribute of command class, which must extend Flarum\Console\AbstractCommand
+     */
+    public function add($command)
```

Comment:
I'm not 100% sure, but for now I would prefer to name this `command`.

We will have to review all extender names for consistency before we release stable. There is no way around that anyway.

Reply:
Done!